src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -16,19 +16,21 @@
 import { JsonConverter } from 'json2typescript';
 import { AbstractConverter, ensureIsArray } from './AbstractConverter';
 import ShapeBpmnElement, { ShapeBpmnEvent } from '../../../../model/bpmn/shape/ShapeBpmnElement';
-import { ShapeBpmnElementKind } from '../../../../model/bpmn/shape/ShapeBpmnElementKind';
+import { ShapeBpmnElementKind, supportedBpmnElementsKinds } from '../../../../model/bpmn/shape/ShapeBpmnElementKind';
 import { Process } from '../Definitions';
 import SequenceFlow from '../../../../model/bpmn/edge/SequenceFlow';
 import Waypoint from '../../../../model/bpmn/edge/Waypoint';
 import { ShapeBpmnEventKind, supportedBpmnEventKinds } from '../../../../model/bpmn/shape/ShapeBpmnEventKind';
 import ShapeUtil from '../../../../model/bpmn/shape/ShapeUtil';
+import { SequenceFlowKind } from '../../../../model/bpmn/edge/SequenceFlowKind';
 
 const convertedFlowNodeBpmnElements: ShapeBpmnElement[] = [];
 const convertedLaneBpmnElements: ShapeBpmnElement[] = [];
 const convertedProcessBpmnElements: ShapeBpmnElement[] = [];
 const convertedSequenceFlows: SequenceFlow[] = [];
+const defaultSequenceFlowIds: string[] = [];
 
-const flowNodeKinds = Object.values(ShapeBpmnElementKind).filter(kind => {
+const flowNodeKinds = supportedBpmnElementsKinds.filter(kind => {
```

Comment:
:question: let's move `flowNodeKinds` to `ShapeUtil` to have all such list at the same place, especially if we keep `supportedBpmnElementsKinds`

Reply:
I wanted to move it, but I can't remember why I didn't do it.
I will do it tomorrow ;)

Reply:
ok, no problem, we left this from previous PR so this is clearly not urgent

Reply:
As we discussed, I'll move it in ShapeUtil

Reply:
Done